src/components/common/SideBar.tsx

```
@@ -2,17 +2,16 @@
 import Image from "next/image";
 import { usePathname, useRouter } from "next/navigation";
 
-import { AlertCircle, Contact2, FileSignature, Headset, Map, Users } from "lucide-react";
+import { AlertCircle, Contact2, Headset, Map, Users } from "lucide-react";
```

Biome (2.4.7)

[error] Do not shadow the global "Map" property.

(lint/suspicious/noShadowRestrictedNames)